### DISCUSSION TITLE
"Type Error: Property 'actionInput' does not exist on type 'PrismaClient' during Vercel Build"

### DISCUSSION THREAD

**Discussion by boobal003** [vercel/community#6950] [Help]
### Summary

Hello,

I am encountering an error during my Vercel build process. The error message is:

Type error: Property 'actionInput' does not exist on type 'PrismaClient<PrismaClientOptions, never, DefaultArgs>'.

./src/app/api/forminput/route.ts:24:31
--
15:42:50.203 | Type error: Property 'actionInput' does not exist on type 'PrismaClient<PrismaClientOptions, never, DefaultArgs>'.
15:42:50.203 |  
15:42:50.204 | 22 \|     const data = validation.data;
15:42:50.204 | 23 \|
15:42:50.204 | > 24 \|     const form = await prisma.actionInput.create({
15:42:50.204 | \|                               ^
15:42:50.204 | 25 \|       data: {
15:42:50.204 | 26 \|         text: data.text,
15:42:50.204 | 27 \|         textarea: data.textarea,
15:42:50.285 | Error: Command "next build && npx prisma generate" exited with 1

schema.prisma:
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
}

model Input {
  id        String   @id @default(auto()) @map("_id") @db.ObjectId
  text      String
  textarea  String
  number    String
  url       String
  email     String
  password  String
  select    String
  radio     String
  check     Boolean
  dob       String
  time      String
  datetime  String
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}

model ClientInput {
  id        String   @id @default(auto()) @map("_id") @db.ObjectId
  text      String
  textarea  String
  number    String
  url       String
  email     String
  password  String
  select    String
  radio     String
  check     Boolean
  dob       String
  time      String
  datetime  String
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}

model ActionInput {
  id              String   @id @default(auto()) @map("_id") @db.ObjectId
  text            String
  textarea        String
  email           String
  number          String
  url             String
  password        String
  confirmPassword String
  select          String
  check           Boolean
  radio           String
  dob             String
  time            String
  datetime        String
  imageUrl        String
  createdAt       DateTime @default(now())
  updatedAt       DateTime @updatedAt
}

I have tried updating my Prisma schema and ensuring that actionInput exists, but I am still facing this issue. it work locally no issue, but i deploy in vercel it show thie error repeatly,  Any help would be greatly appreciated!

### Steps to Reproduce

![Capture](https://github.com/vercel/community/assets/141622627/47952e84-64aa-4965-954c-10516bc19949)

**Comment by versecafe** [comment]
@boobal003 Are you sure your prisma files are being generated on Vercel in your build step?
